### DISCUSSION TITLE
Queue Triggered Job Dequeue

### DISCUSSION THREAD

**Discussion by erenhakan34** [Azure/azure-functions-durable-extension#2163] [Q&A]
Hello,

I have a queue triggered function(not durable function) that takes only a string message. If my function fails due to some business reason(it's normal to fail), I want my function not to re-process the message over and over after certain timeout. How can I get it done?

Thanks.

**Comment by davidmrdavid** [comment]
Hi @erenhakan34, thanks for reaching out.

I think you could potentially implement this with a Durable Functions orchestrator that uses `callActivityWithRetry` and sets the `maxRetryIntervalInMilliseconds` option to be the timeout internal after which you won't retry the operation anymore. You can read more about our retry-options here: https://docs.microsoft.com/en-us/javascript/api/durable-functions/retryoptions?view=azure-node-latest#durable-functions-retryoptions-maxretryintervalinmilliseconds
